### PR TITLE
Fix REPL hanging

### PR DESCRIPTION
**Pull request by drwpow**
## Fix REPL

The REPL wasn’t working because of a mismatch with `shiki`. We were loading the latest version of `shiki` from `unpkg`, which mismatched with our locally-loaded version. This changes the REPL to use a local copy of `shiki`, which will not only prevent the problem in the future, but also allow our REPL to not depend on unpkg to work!

## Fix Firefox

The REPL hasn’t supported Firefox for some time, but the error was obtuse. A better error was added:

**Before**

Screen Shot 2021-12-02 at 15 55 11

**After**
Screen Shot 2021-12-02 at 15 56 11

A good advantage to this strategy is we won’t have to keep this updated! If Firefox ever adds support, this should just start working without requiring an update from us.

### CODE REVIEW

File: src/editor/workers/build.ts [hunk 1]
```diff
@@ -60,262 +60,266 @@ let _initialized = false;
...
+  try {
+    if (!_initialized) {
+      await EsbuildInitialize({
+        worker: false,
+        wasmURL: `/play/esbuild.wasm`,
+      });
+      await AstroInitialize({
+        wasmURL: '/play/astro.wasm',
+      });
+      setCDN('/play/shiki/');
```
Comment (drwpow) [inline src/editor/workers/build.ts:72]: Change 1: use local `shiki`

File: src/editor/workers/build.ts [hunk 2]
```diff
@@ -60,262 +60,266 @@ let _initialized = false;
...
+              type: `Build complete`,
+              values: files,
+              js,
+              html,
+              shiki,
+            },
+          });
+        } catch (error) {
+          // Firefox error
+          if (String(error).includes('Dynamic module import is disabled or not supported in this context')) {
```
Comment (drwpow) [inline src/editor/workers/build.ts:285]: Change 2: Firefox error

**Review by jonathantneal — APPROVED** [review]
I think I get it. Hard to see with all the styling changes. 😅

**Comment by drwpow** [comment]
> Hard to see with all the styling changes.

Ah, sorry. Yes this repo needs a format! Will merge this and do that in a followup to reduce noise.

**Pull request merged**